src/status_im/contexts/profile/edit/header/events.cljs

```diff
@@ -13,45 +13,51 @@
           (update db :profile/profile dissoc :images))}))
 
 (rf/reg-event-fx :profile/edit-profile-picture-success
- (fn [_ [images]]
-   (let [has-picture? (rf/sub [:profile/has-picture])]
+ (fn [{db :db} [show-toast? images]]
```

Comment:
Maybe this first argument should also be a map?

I ask because it might read better to use a map instead of boolean, for example when we test the code:
```
[:profile/delete-profile-picture-success true]
```
would become
```
[:profile/delete-profile-picture-success {:show-toast? true}]
```
which I think is more descriptive of what's being enabled. thoughts?

Reply:
I agree with you @seanstrom I updated them now.

I passed single arguments because these events were only used once and internally as follow-up events. 

Sometimes I'm a bit concerned about creating a new CLJS map and then destructuring it in another function when the API is not widely usedm I know it improves the reading but I think it's not that hard to understand. Usually for functions/events rarely used and with few args (e.g. 3) sometimes I prefer this approach.